IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md

```diff
@@ -0,0 +1 @@
+Number of phase that the equipment operates on.
```

Comment:
this has a suggestion on another pull request needs to be harmonised.

Reply:
Done. Commit : 6be975f2f1a62ccae3a2de607523c16a192995d1